python/runner.n

```diff
@@ -0,0 +1,3 @@
+print "test"
+let eeeeee:int = 0
```

Comment:
We should have a way of explicitly declaring what should be exported. I agree with Rust that things should be private by default. I quite prefer simply tagging `export`/`public`/`pub` to something than listing all the exports at the end/top of the file, so perhaps

```js
export let eeeeee: int = 0
public let eeeeee: int = 0
pub let eeeeee: int = 0
```

Go makes capital variable names exported, but I'm not a fan of that